connector/docs/config.md

```diff
@@ -0,0 +1,84 @@
+# Configuring Salesforce Operations
+
+[[Importing the Salesforce Certificate]](#importing-the-salesforce-certificate)  [[Common parameters]](#common-parameters) [[Salesforce Operations]](#salesforce-perations) [[Logging out of Salesforce]](#logging-out-of-salesforce)
+
+## Importing the Salesforce Certificate
+
+To use the Salesforce connector, add the  <salesforce.init>  element in your configuration before carrying out any other Salesforce operations.
+
+Before you start configuring the Salesforce connector, make sure to import the Salesforce certificate to your ESB client keystore.
+
+* Follow the steps below to import the Salesforce certificate into the ESB client keystore:
+
+    1. Log in to your Salesforce account in your browser (e.g., https://login.salesforce.com), and click the lock on the address bar to view the certificate.
+    2. Export the certificate to the file system.
+    3. Import the certificate into the ESB client keystore using the Management Console or by using the following command:
+    ```
+    keytool -importcert -file <certificate file> -keystore <ESB>/repository/resources/security/client-truststore.jks -alias "Salesforce"
+    ```
+    4. Restart the server and deploy the following Salesforce configuration:
+
+**init**
+```xml
+<salesforce.init>
+    <username>MyUsername</username>
+    <password>MyPassword</password>
+    <loginUrl>https://login.salesforce.com/services/Soap/u/27.0</loginUrl>
+    <blocking>false</blocking>
+</salesforce.init>
+```
+**Properties** 
+* username:  The user name to access the Salesforce account.
+* password:  The password provided here is a concatenation of the user password and the security token provided by Salesforce.
+* loginUrl:  The login URL to access the Salesforce account.
+* blocking:  Indicated whether the connector needs to perform blocking invocations to Salesforce. (Supported in ESB 4.9.0 and later) 
+
+```text
+*  Users can obtain the security token by changing the password or resetting the security token using the Salesforce user interface. The new security token will be sent to the email address recorded in the user's Salesforce record.
+*  The response of this operation is attached to the message body and is used for subsequent Salesforce operations.
+*  The session ID is saved into the property salesforce.sessionId, and the server URL is saved into salesforce.serviceUrl. If the given login details are invalid, the specified fault sequence will be triggered.
```

Comment:
The session ID is saved in the property salesforce.sessionId and the server URL is saved in salesforce.serviceUrl. If the given login details are invalid, the specified fault sequence is triggered.